alvr/adb/src/lib.rs

```diff
@@ -49,16 +50,39 @@ impl WiredConnection {
             );
         }
 
-        #[cfg(debug_assertions)]
-        let process_name = "alvr.client.dev";
-        #[cfg(not(debug_assertions))]
-        let process_name = "alvr.client.stable";
+        let process_name = match &settings.connection.client_flavor {
+            ClientFlavor::Store => if alvr_common::is_stable() {
+                "alvr.client"
+            } else {
+                "alvr.client.dev"
+            }
+            .to_owned(),
+            ClientFlavor::Github => if alvr_common::is_stable() {
+                "alvr.client.stable"
+            } else {
+                "alvr.client.dev"
+            }
+            .to_owned(),
+            ClientFlavor::Custom(name) => name.clone(),
+        };
```

Comment:
This is missing the fallbacks.
Store: store -> gh
GitHub: gh -> store
Custom: custom -> store -> gh

Skip store if not stable. 

You can implement this by storing a slice to `process_names`. At this point it would be better to extract the package names to string constants.